lib/cronofy/api_key.rb

```diff
@@ -15,6 +15,7 @@ def initialize(client, client_secret)
     # @see Client#request
     def request(verb, path, opts = {}, &block)
       configure_authentication!(opts)
+      opts = { snaky: false }.merge(opts)
```

Comment:
This was necessary to avoid warning logs being output. Our Smart Invite API responses include a key of `method`, which caused the following warning log:

> WARN -- : You are setting a key that conflicts with a built-in method SnakyHash::StringKeyed#method defined in Kernel. This can cause unexpected behavior when accessing the key as a property. You can still access the key via the #[] method.

Seems like an oversight on the gem's part, but as far as I can see, we don't want the snake-case-ifying behaviour which was introduced in v2 as per https://gitlab.com/oauth-xx/oauth2/-/blob/main/README.md